### PR TITLE
Change variable names: filenames to paths

### PR DESCRIPTION
**Pull request by yuyu2172**
Related #392

### CODE REVIEW

File: chainercv/datasets/camvid/camvid_dataset.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def get_example(self, i):
         """
         if i >= len(self):
             raise IndexError('index is too large')
-        img_filename, label_filename = self.filenames[i]
+        img_filename, label_filename = self.paths[i]
```
Comment (Hakuyume) [inline chainercv/datasets/camvid/camvid_dataset.py]: `img_filename` -> `img_path`
`label_filename` -> `label_path`

File: chainercv/datasets/camvid/camvid_dataset.py [hunk 2]
```diff
@@ -83,12 +83,12 @@ def __init__(self, data_dir='auto', split='train'):
             data_dir = get_camvid()
 
         img_list_filename = os.path.join(data_dir, '{}.txt'.format(split))
```
Comment (Hakuyume) [inline chainercv/datasets/camvid/camvid_dataset.py]: `img_list_filename` -> `img_list_path`

**Comment by Hakuyume** [comment]
I found some `filename` s in other part than `datasets`. How do you think of them? I think some of them should be kept for the consistency with chainer.

**Comment by yuyu2172** [comment]
You are right.
I think it is OK to leave them as is for consistency.

**Pull request merged**